packages/ilib-loctool-webos-dart/DartFile.js

```diff
@@ -116,11 +116,12 @@ DartFile.prototype.makeKey = function(source) {
 translate("High")
 translate("High", key: "home_connect_210") // will be supported for webos
 translate("{arg1} app cannot be deleted.", arg:{"arg1": "Settings"})
+translate("The lowest temp is {arg1}", args: <String, int>{"arg1": 15})
 translate("The lowest temperature is {arg1} and the highest temperature is {arg2}.", arg:{"arg1": 15, "arg2": 30})
 */
 var reTranslate = new RegExp(/translate\s*\(\s*("((\\"|[^"])*)"|'((\\'|[^'])*)')\s*\)/g);
 var reTranslateWithKey = new RegExp(/translate\s*\(\s*("((\\"|[^"])*)"|'((\\'|[^'])*)')\s*\,\s*(key)\s*\:\s*("((\\"|[^"])*)"|'((\\'|[^'])*)')\s*\)/g);
-var reTranslateWithArg = new RegExp(/translate\s*\(\s*("((\\"|[^"])*)"|'((\\'|[^'])*)')\s*\,\s*args\s*\:\s*\{(.*)\}\s*\)/g);
+var reTranslateWithArg = new RegExp(/translate\s*\(\s*("((\\"|[^"])*)"|'((\\'|[^'])*)')\s*\,\s*args\s*\:\s*(\<.*\>)?\s*\{(.*)\}\s*\)/g);
```

Comment:
```suggestion
var reTranslateWithArg = new RegExp(/translate\s*\(\s*("((\\"|[^"])*)"|'((\\'|[^'])*)')\s*\,\s*args\s*\:/g);
```

Reply:
I updated it. I didn't change the reTranslationWithKey expression in sameway. because it needs to extract the key value, which is different from args.